sdmetrics/reports/single_table/_properties/column_pair_trends.py

```diff
@@ -281,23 +237,22 @@ def _generate_details(self, real_data, synthetic_data, metadata, progress_bar):
 
                 continue
 
-            metric = self._get_metric(sdtype_col_1, sdtype_col_2)
+            columns_real, metric = self._get_columns_data_and_metric(
+                column_name_1, column_name_2, processed_real_data, discrete_real, metadata
+            )
+            columns_synthetic, _ = self._get_columns_data_and_metric(
+                column_name_1, column_name_2, processed_synthetic_data,
+                discrete_synthetic, metadata
```

Comment:
Why not get both the synthetic and real data at once?

Reply:
Yes done in [582456b](https://github.com/sdv-dev/SDMetrics/pull/368/commits/582456bbfd5ac86cd5f861c4a420900ea6fa27a0)